pkg/pillar/types/volumetypes.go

```diff
@@ -398,18 +420,30 @@ type VolumeRefStatus struct {
 	VerifyOnly             bool
 	Target                 zconfig.Target
 	CustomMeta             string
+	// kube pod container image
```

Comment:
What is this? Comment doesn't explain.

Reply:
Is is still there. But it wouldn't hurt to explain what ReferenceName is.

Reply:
I will remove the comment but keep the ReferenceName.  We actually copy the VolumeStatus.ReferenceName to VolumeRefStatus.ReferenceName.  If you want to change then we need to change VolumeStatus too and all over the code.